sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3317,6 +3317,19 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
         """.stripMargin).collect()
     }
   }
+
+  test("SPARK-29595: Unorder insertion in NAMED_STRUCT should throw exception") {
+    sql("CREATE TABLE tb1 USING PARQUET AS (SELECT NAMED_STRUCT('a', 1, 'b', 2) AS data)")
+    val error = intercept[AnalysisException] {
+      sql("INSERT INTO tb1 VALUES NAMED_STRUCT('b', 4, 'a', 3)")
+    }.getMessage
+    assert(error contains
```

Comment:
nit: format `error.contains(..)` along with the other ones in this file.

Reply:
Fixed.